ui/yarn.lock

```diff
@@ -12358,7 +12386,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"ember-auto-import@npm:2.6.3, ember-auto-import@npm:^2.4.2, ember-auto-import@npm:^2.4.3, ember-auto-import@npm:^2.5.0":
+"ember-auto-import@npm:2.6.3, ember-auto-import@npm:^2.4.2, ember-auto-import@npm:^2.4.3, ember-auto-import@npm:^2.5.0, ember-auto-import@npm:^2.6.3":
```

Comment:
Running the upgrade command added `ember-auto-import@npm:^2.6.3` but there is already `ember-auto-import@npm:2.6.3` (without the ^), so I'm not sure this is necessary. I didn't want to manually modify anything though, so this is just the result of running `yarn up @hashicorp/design-system-components` (from [this HDS website page](https://hds-website-hashicorp.vercel.app/getting-started/for-engineers))

Reply:
It looks like this is the [version specified by the HDS components package](https://github.com/hashicorp/design-system/blob/main/packages/components/package.json#L48). You had the right instincts to not modify the lock file. Since it is [generated automatically by yarn it shouldn't be manually changed](https://classic.yarnpkg.com/lang/en/docs/yarn-lock/#toc-managed-by-yarn) and any dependency changes should be done in the `package.json` file or with the yarn cli.

Reply:
Thank you for explaining that!